src/pkcs/clu_pkcs12.c

```diff
@@ -129,6 +132,11 @@ int wolfCLU_PKCS12(int argc, char** argv)
         }
     }
 
+    /* with only supporting parsing, an input file is expected */
```

Comment:
I also don't understand this comment, at least the first "with only supporting parsing" part.